### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by hollandjg** [AutoResearch/autora#424] [General Announcements]
## What's Changed
* docs: update documentation building script to use poetry for installing dependencies by @hollandjg in https://github.com/AutoResearch/autora/pull/344
* chore(deps-dev): bump mkdocs-material from 9.1.5 to 9.1.6 by @dependabot in https://github.com/AutoResearch/autora/pull/345
* chore(deps-dev): bump pytest from 7.2.2 to 7.3.0 by @dependabot in https://github.com/AutoResearch/autora/pull/346
* docs: adjusted mkdocs theme by @musslick in https://github.com/AutoResearch/autora/pull/348
* doc/revised overview by @musslick in https://github.com/AutoResearch/autora/pull/351
* chore(deps-dev): bump pytest from 7.3.0 to 7.3.1 by @dependabot in https://github.com/AutoResearch/autora/pull/352
* refactor: remove outdated plot_utils for darts by @hollandjg in https://github.com/AutoResearch/autora/pull/350
* chore(deps): bump pandas from 2.0.0 to 2.0.1 by @dependabot in https://github.com/AutoResearch/autora/pull/353
* chore(deps-dev): bump mkdocs-material from 9.1.6 to 9.1.7 by @dependabot in https://github.com/AutoResearch/autora/pull/354
* chore(deps): bump imageio from 2.27.0 to 2.28.0 by @dependabot in https://github.com/AutoResearch/autora/pull/356
* chore(deps): bump numpy from 1.24.2 to 1.24.3 by @dependabot in https://github.com/AutoResearch/autora/pull/355
* chore(deps-dev): bump mkdocs-material from 9.1.7 to 9.1.8 by @dependabot in https://github.com/AutoResearch/autora/pull/377
* chore(deps-dev): bump ipython from 8.12.0 to 8.12.1 by @dependabot in https://github.com/AutoResearch/autora/pull/378
* chore(deps-dev): bump mkdocs-gen-files from 0.4.0 to 0.5.0 by @dependabot in https://github.com/AutoResearch/autora/pull/379
* chore(deps): bump imageio from 2.28.0 to 2.28.1 by @dependabot in https://github.com/AutoResearch/autora/pull/380
* chore(deps-dev): bump mkdocs-material from 9.1.8 to 9.1.10 by @dependabot in https://github.com/AutoResearch/autora/pull/390
* chore(deps-dev): bump pre-commit from 3.2.2 to 3.3.1 by @dependabot in https://github.com/AutoResearch/autora/pull/391
* chore(deps-dev): bump mkdocstrings-python from 0.9.0 to 0.10.1 by @dependabot in https://github.com/AutoResearch/autora/pull/392
* chore(deps-dev): bump ipython from 8.12.1 to 8.12.2 by @dependabot in https://github.com/AutoResearch/autora/pull/393
* chore(deps-dev): bump mkdocs-material from 9.1.10 to 9.1.12 by @dependabot in https://github.com/AutoResearch/autora/pull/404
* chore(deps-dev): bump mkdocstrings-python from 0.10.1 to 1.0.0 by @dependabot in https://github.com/AutoResearch/autora/pull/405
* chore(deps): bump pymdown-extensions from 9.10 to 10.0 by @dependabot in https://github.com/AutoResearch/autora/pull/407
* chore(deps): bump sympy from 1.11.1 to 1.12 by @dependabot in https://github.com/AutoResearch/autora/pull/406
* chore(deps-dev): bump pre-commit from 3.3.1 to 3.3.2 by @dependabot in https://github.com/AutoResearch/autora/pull/411
* chore(deps-dev): bump mkdocs-material from 9.1.12 to 9.1.14 by @dependabot in https://github.com/AutoResearch/autora/pull/412
* chore(deps): bump requests from 2.28.2 to 2.31.0 by @dependabot in https://github.com/AutoResearch/autora/pull/413
* chore(deps): bump tornado from 6.2 to 6.3.2 by @dependabot in https://github.com/AutoResearch/autora/pull/418
* chore(deps): bump imageio from 2.28.1 to 2.30.0 by @dependabot in https://github.com/AutoResearch/autora/pull/419
* chore(deps-dev): bump mkdocstrings-python from 1.0.0 to 1.1.0 by @dependabot in https://github.com/AutoResearch/autora/pull/420
* chore(deps): bump pandas from 2.0.1 to 2.0.2 by @dependabot in https://github.com/AutoResearch/autora/pull/421
* refactor!: convert this into the "parent" package for autora, without code by @hollandjg in https://github.com/AutoResearch/autora/pull/371

**Full Changelog**: https://github.com/AutoResearch/autora/compare/v2.4.1...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AutoResearch/autora/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by chadcwilliams** [comment]
New version 4.0.0 out now!
